[[0, 0, 0], 5, {"started": 1559151964.1238267, "submitted": 1559151964.123734, "finished": 1559151978.4759312}, null, "Traceback (most recent call last):\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/hpbandster/core/worker.py\", line 206, in start_computation\n    result = {'result': self.compute(*args, config_id=id, **kwargs),\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 475, in compute\n    _ = classifier.train(input_fn_train)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 363, in train\n    loss = self._train_model(input_fn, hooks, saving_listeners)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 843, in _train_model\n    return self._train_model_default(input_fn, hooks, saving_listeners)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 856, in _train_model_default\n    features, labels, model_fn_lib.ModeKeys.TRAIN, self.config)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 831, in _call_model_fn\n    model_fn_results = self._model_fn(features=features, **kwargs)\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 139, in __call__\n    train_op = self.create_train_op(model) if mode == tf.estimator.ModeKeys.TRAIN else None\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 190, in create_train_op\n    if model.config['lr_scheduler'] in [\"inv_exp_fast\", \"inv_exp_slow\"]:\nKeyError: 'lr_scheduler'\n"]
[[1, 0, 0], 5, {"started": 1559151978.479701, "submitted": 1559151978.4794996, "finished": 1559151978.9906025}, null, "Traceback (most recent call last):\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/hpbandster/core/worker.py\", line 206, in start_computation\n    result = {'result': self.compute(*args, config_id=id, **kwargs),\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 475, in compute\n    _ = classifier.train(input_fn_train)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 363, in train\n    loss = self._train_model(input_fn, hooks, saving_listeners)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 843, in _train_model\n    return self._train_model_default(input_fn, hooks, saving_listeners)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 856, in _train_model_default\n    features, labels, model_fn_lib.ModeKeys.TRAIN, self.config)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 831, in _call_model_fn\n    model_fn_results = self._model_fn(features=features, **kwargs)\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 139, in __call__\n    train_op = self.create_train_op(model) if mode == tf.estimator.ModeKeys.TRAIN else None\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 190, in create_train_op\n    if model.config['lr_scheduler'] in [\"inv_exp_fast\", \"inv_exp_slow\"]:\nKeyError: 'lr_scheduler'\n"]
[[2, 0, 0], 5, {"started": 1559151978.9941635, "submitted": 1559151978.9939857, "finished": 1559151979.4865553}, null, "Traceback (most recent call last):\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/framework/ops.py\", line 1567, in _create_c_op\n    c_op = c_api.TF_FinishOperation(op_desc)\ntensorflow.python.framework.errors_impl.InvalidArgumentError: Negative dimension size caused by subtracting 5 from 3 for 'ConvNet_global_view_0/max_pooling1d_4/MaxPool' (op: 'MaxPool') with input shapes: [?,1,3,512].\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/hpbandster/core/worker.py\", line 206, in start_computation\n    result = {'result': self.compute(*args, config_id=id, **kwargs),\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 475, in compute\n    _ = classifier.train(input_fn_train)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 363, in train\n    loss = self._train_model(input_fn, hooks, saving_listeners)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 843, in _train_model\n    return self._train_model_default(input_fn, hooks, saving_listeners)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 856, in _train_model_default\n    features, labels, model_fn_lib.ModeKeys.TRAIN, self.config)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 831, in _call_model_fn\n    model_fn_results = self._model_fn(features=features, **kwargs)\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 137, in __call__\n    model = self._model_class(features, labels, config, mode)\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 249, in __init__\n    self.build()\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 344, in build\n    cnn_layers = self.build_cnn_layers()\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 290, in build_cnn_layers\n    net = tf.layers.max_pooling1d(inputs=net, pool_size=pool_size, strides=self.config['pool_stride'])\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/layers/pooling.py\", line 231, in max_pooling1d\n    return layer.apply(inputs)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/layers/base.py\", line 828, in apply\n    return self.__call__(inputs, *args, **kwargs)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/layers/base.py\", line 717, in __call__\n    outputs = self.call(inputs, *args, **kwargs)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/layers/pooling.py\", line 86, in call\n    data_format=data_format)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/ops/nn_ops.py\", line 2142, in max_pool\n    name=name)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/ops/gen_nn_ops.py\", line 4604, in max_pool\n    data_format=data_format, name=name)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/framework/op_def_library.py\", line 787, in _apply_op_helper\n    op_def=op_def)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/framework/ops.py\", line 3392, in create_op\n    op_def=op_def)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/framework/ops.py\", line 1734, in __init__\n    control_input_ops)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/framework/ops.py\", line 1570, in _create_c_op\n    raise ValueError(str(e))\nValueError: Negative dimension size caused by subtracting 5 from 3 for 'ConvNet_global_view_0/max_pooling1d_4/MaxPool' (op: 'MaxPool') with input shapes: [?,1,3,512].\n"]
[[3, 0, 0], 5, {"started": 1559151979.490067, "submitted": 1559151979.4899127, "finished": 1559151980.229333}, null, "Traceback (most recent call last):\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/hpbandster/core/worker.py\", line 206, in start_computation\n    result = {'result': self.compute(*args, config_id=id, **kwargs),\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 475, in compute\n    _ = classifier.train(input_fn_train)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 363, in train\n    loss = self._train_model(input_fn, hooks, saving_listeners)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 843, in _train_model\n    return self._train_model_default(input_fn, hooks, saving_listeners)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 856, in _train_model_default\n    features, labels, model_fn_lib.ModeKeys.TRAIN, self.config)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 831, in _call_model_fn\n    model_fn_results = self._model_fn(features=features, **kwargs)\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 139, in __call__\n    train_op = self.create_train_op(model) if mode == tf.estimator.ModeKeys.TRAIN else None\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 190, in create_train_op\n    if model.config['lr_scheduler'] in [\"inv_exp_fast\", \"inv_exp_slow\"]:\nKeyError: 'lr_scheduler'\n"]
[[4, 0, 0], 5, {"started": 1559151980.2327726, "submitted": 1559151980.232623, "finished": 1559151980.6841097}, null, "Traceback (most recent call last):\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/hpbandster/core/worker.py\", line 206, in start_computation\n    result = {'result': self.compute(*args, config_id=id, **kwargs),\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 475, in compute\n    _ = classifier.train(input_fn_train)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 363, in train\n    loss = self._train_model(input_fn, hooks, saving_listeners)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 843, in _train_model\n    return self._train_model_default(input_fn, hooks, saving_listeners)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 856, in _train_model_default\n    features, labels, model_fn_lib.ModeKeys.TRAIN, self.config)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 831, in _call_model_fn\n    model_fn_results = self._model_fn(features=features, **kwargs)\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 139, in __call__\n    train_op = self.create_train_op(model) if mode == tf.estimator.ModeKeys.TRAIN else None\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 190, in create_train_op\n    if model.config['lr_scheduler'] in [\"inv_exp_fast\", \"inv_exp_slow\"]:\nKeyError: 'lr_scheduler'\n"]
[[5, 0, 0], 5, {"started": 1559151980.6876304, "submitted": 1559151980.687468, "finished": 1559151981.111685}, null, "Traceback (most recent call last):\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/hpbandster/core/worker.py\", line 206, in start_computation\n    result = {'result': self.compute(*args, config_id=id, **kwargs),\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 475, in compute\n    _ = classifier.train(input_fn_train)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 363, in train\n    loss = self._train_model(input_fn, hooks, saving_listeners)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 843, in _train_model\n    return self._train_model_default(input_fn, hooks, saving_listeners)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 856, in _train_model_default\n    features, labels, model_fn_lib.ModeKeys.TRAIN, self.config)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 831, in _call_model_fn\n    model_fn_results = self._model_fn(features=features, **kwargs)\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 139, in __call__\n    train_op = self.create_train_op(model) if mode == tf.estimator.ModeKeys.TRAIN else None\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 190, in create_train_op\n    if model.config['lr_scheduler'] in [\"inv_exp_fast\", \"inv_exp_slow\"]:\nKeyError: 'lr_scheduler'\n"]
[[6, 0, 0], 5, {"started": 1559151981.1152992, "submitted": 1559151981.1151216, "finished": 1559151981.8538795}, null, "Traceback (most recent call last):\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/hpbandster/core/worker.py\", line 206, in start_computation\n    result = {'result': self.compute(*args, config_id=id, **kwargs),\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 475, in compute\n    _ = classifier.train(input_fn_train)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 363, in train\n    loss = self._train_model(input_fn, hooks, saving_listeners)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 843, in _train_model\n    return self._train_model_default(input_fn, hooks, saving_listeners)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 856, in _train_model_default\n    features, labels, model_fn_lib.ModeKeys.TRAIN, self.config)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 831, in _call_model_fn\n    model_fn_results = self._model_fn(features=features, **kwargs)\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 139, in __call__\n    train_op = self.create_train_op(model) if mode == tf.estimator.ModeKeys.TRAIN else None\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 190, in create_train_op\n    if model.config['lr_scheduler'] in [\"inv_exp_fast\", \"inv_exp_slow\"]:\nKeyError: 'lr_scheduler'\n"]
[[7, 0, 0], 5, {"started": 1559151981.8574796, "submitted": 1559151981.8573933, "finished": 1559151982.2544405}, null, "Traceback (most recent call last):\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/hpbandster/core/worker.py\", line 206, in start_computation\n    result = {'result': self.compute(*args, config_id=id, **kwargs),\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 475, in compute\n    _ = classifier.train(input_fn_train)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 363, in train\n    loss = self._train_model(input_fn, hooks, saving_listeners)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 843, in _train_model\n    return self._train_model_default(input_fn, hooks, saving_listeners)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 856, in _train_model_default\n    features, labels, model_fn_lib.ModeKeys.TRAIN, self.config)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 831, in _call_model_fn\n    model_fn_results = self._model_fn(features=features, **kwargs)\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 139, in __call__\n    train_op = self.create_train_op(model) if mode == tf.estimator.ModeKeys.TRAIN else None\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 190, in create_train_op\n    if model.config['lr_scheduler'] in [\"inv_exp_fast\", \"inv_exp_slow\"]:\nKeyError: 'lr_scheduler'\n"]
[[8, 0, 0], 5, {"started": 1559151982.2615254, "submitted": 1559151982.2614393, "finished": 1559151982.8219693}, null, "Traceback (most recent call last):\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/hpbandster/core/worker.py\", line 206, in start_computation\n    result = {'result': self.compute(*args, config_id=id, **kwargs),\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 475, in compute\n    _ = classifier.train(input_fn_train)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 363, in train\n    loss = self._train_model(input_fn, hooks, saving_listeners)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 843, in _train_model\n    return self._train_model_default(input_fn, hooks, saving_listeners)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 856, in _train_model_default\n    features, labels, model_fn_lib.ModeKeys.TRAIN, self.config)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 831, in _call_model_fn\n    model_fn_results = self._model_fn(features=features, **kwargs)\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 139, in __call__\n    train_op = self.create_train_op(model) if mode == tf.estimator.ModeKeys.TRAIN else None\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 190, in create_train_op\n    if model.config['lr_scheduler'] in [\"inv_exp_fast\", \"inv_exp_slow\"]:\nKeyError: 'lr_scheduler'\n"]
[[9, 0, 0], 5, {"started": 1559151982.8253014, "submitted": 1559151982.8252196, "finished": 1559151983.3613822}, null, "Traceback (most recent call last):\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/hpbandster/core/worker.py\", line 206, in start_computation\n    result = {'result': self.compute(*args, config_id=id, **kwargs),\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 475, in compute\n    _ = classifier.train(input_fn_train)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 363, in train\n    loss = self._train_model(input_fn, hooks, saving_listeners)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 843, in _train_model\n    return self._train_model_default(input_fn, hooks, saving_listeners)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 856, in _train_model_default\n    features, labels, model_fn_lib.ModeKeys.TRAIN, self.config)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 831, in _call_model_fn\n    model_fn_results = self._model_fn(features=features, **kwargs)\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 139, in __call__\n    train_op = self.create_train_op(model) if mode == tf.estimator.ModeKeys.TRAIN else None\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 190, in create_train_op\n    if model.config['lr_scheduler'] in [\"inv_exp_fast\", \"inv_exp_slow\"]:\nKeyError: 'lr_scheduler'\n"]
[[10, 0, 0], 5, {"started": 1559151983.3648503, "submitted": 1559151983.364769, "finished": 1559151984.1493912}, null, "Traceback (most recent call last):\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/hpbandster/core/worker.py\", line 206, in start_computation\n    result = {'result': self.compute(*args, config_id=id, **kwargs),\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 475, in compute\n    _ = classifier.train(input_fn_train)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 363, in train\n    loss = self._train_model(input_fn, hooks, saving_listeners)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 843, in _train_model\n    return self._train_model_default(input_fn, hooks, saving_listeners)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 856, in _train_model_default\n    features, labels, model_fn_lib.ModeKeys.TRAIN, self.config)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 831, in _call_model_fn\n    model_fn_results = self._model_fn(features=features, **kwargs)\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 139, in __call__\n    train_op = self.create_train_op(model) if mode == tf.estimator.ModeKeys.TRAIN else None\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 190, in create_train_op\n    if model.config['lr_scheduler'] in [\"inv_exp_fast\", \"inv_exp_slow\"]:\nKeyError: 'lr_scheduler'\n"]
[[11, 0, 0], 5, {"started": 1559151984.152867, "submitted": 1559151984.1527855, "finished": 1559151984.796755}, null, "Traceback (most recent call last):\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/hpbandster/core/worker.py\", line 206, in start_computation\n    result = {'result': self.compute(*args, config_id=id, **kwargs),\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 475, in compute\n    _ = classifier.train(input_fn_train)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 363, in train\n    loss = self._train_model(input_fn, hooks, saving_listeners)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 843, in _train_model\n    return self._train_model_default(input_fn, hooks, saving_listeners)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 856, in _train_model_default\n    features, labels, model_fn_lib.ModeKeys.TRAIN, self.config)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 831, in _call_model_fn\n    model_fn_results = self._model_fn(features=features, **kwargs)\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 139, in __call__\n    train_op = self.create_train_op(model) if mode == tf.estimator.ModeKeys.TRAIN else None\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 190, in create_train_op\n    if model.config['lr_scheduler'] in [\"inv_exp_fast\", \"inv_exp_slow\"]:\nKeyError: 'lr_scheduler'\n"]
[[12, 0, 0], 5, {"started": 1559151984.8005648, "submitted": 1559151984.8004751, "finished": 1559151985.2568743}, null, "Traceback (most recent call last):\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/hpbandster/core/worker.py\", line 206, in start_computation\n    result = {'result': self.compute(*args, config_id=id, **kwargs),\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 475, in compute\n    _ = classifier.train(input_fn_train)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 363, in train\n    loss = self._train_model(input_fn, hooks, saving_listeners)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 843, in _train_model\n    return self._train_model_default(input_fn, hooks, saving_listeners)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 856, in _train_model_default\n    features, labels, model_fn_lib.ModeKeys.TRAIN, self.config)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 831, in _call_model_fn\n    model_fn_results = self._model_fn(features=features, **kwargs)\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 139, in __call__\n    train_op = self.create_train_op(model) if mode == tf.estimator.ModeKeys.TRAIN else None\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 190, in create_train_op\n    if model.config['lr_scheduler'] in [\"inv_exp_fast\", \"inv_exp_slow\"]:\nKeyError: 'lr_scheduler'\n"]
[[13, 0, 0], 5, {"started": 1559151985.260351, "submitted": 1559151985.2602673, "finished": 1559151986.0825872}, null, "Traceback (most recent call last):\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/hpbandster/core/worker.py\", line 206, in start_computation\n    result = {'result': self.compute(*args, config_id=id, **kwargs),\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 475, in compute\n    _ = classifier.train(input_fn_train)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 363, in train\n    loss = self._train_model(input_fn, hooks, saving_listeners)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 843, in _train_model\n    return self._train_model_default(input_fn, hooks, saving_listeners)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 856, in _train_model_default\n    features, labels, model_fn_lib.ModeKeys.TRAIN, self.config)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 831, in _call_model_fn\n    model_fn_results = self._model_fn(features=features, **kwargs)\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 139, in __call__\n    train_op = self.create_train_op(model) if mode == tf.estimator.ModeKeys.TRAIN else None\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 190, in create_train_op\n    if model.config['lr_scheduler'] in [\"inv_exp_fast\", \"inv_exp_slow\"]:\nKeyError: 'lr_scheduler'\n"]
[[14, 0, 0], 5, {"started": 1559151986.0862055, "submitted": 1559151986.086108, "finished": 1559151986.546191}, null, "Traceback (most recent call last):\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/hpbandster/core/worker.py\", line 206, in start_computation\n    result = {'result': self.compute(*args, config_id=id, **kwargs),\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 475, in compute\n    _ = classifier.train(input_fn_train)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 363, in train\n    loss = self._train_model(input_fn, hooks, saving_listeners)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 843, in _train_model\n    return self._train_model_default(input_fn, hooks, saving_listeners)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 856, in _train_model_default\n    features, labels, model_fn_lib.ModeKeys.TRAIN, self.config)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 831, in _call_model_fn\n    model_fn_results = self._model_fn(features=features, **kwargs)\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 139, in __call__\n    train_op = self.create_train_op(model) if mode == tf.estimator.ModeKeys.TRAIN else None\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 190, in create_train_op\n    if model.config['lr_scheduler'] in [\"inv_exp_fast\", \"inv_exp_slow\"]:\nKeyError: 'lr_scheduler'\n"]
[[15, 0, 0], 5, {"started": 1559151986.5502534, "submitted": 1559151986.5501618, "finished": 1559151986.8674078}, null, "Traceback (most recent call last):\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/hpbandster/core/worker.py\", line 206, in start_computation\n    result = {'result': self.compute(*args, config_id=id, **kwargs),\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 475, in compute\n    _ = classifier.train(input_fn_train)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 363, in train\n    loss = self._train_model(input_fn, hooks, saving_listeners)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 843, in _train_model\n    return self._train_model_default(input_fn, hooks, saving_listeners)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 856, in _train_model_default\n    features, labels, model_fn_lib.ModeKeys.TRAIN, self.config)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 831, in _call_model_fn\n    model_fn_results = self._model_fn(features=features, **kwargs)\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 139, in __call__\n    train_op = self.create_train_op(model) if mode == tf.estimator.ModeKeys.TRAIN else None\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 190, in create_train_op\n    if model.config['lr_scheduler'] in [\"inv_exp_fast\", \"inv_exp_slow\"]:\nKeyError: 'lr_scheduler'\n"]
[[16, 0, 0], 5, {"started": 1559151986.8712492, "submitted": 1559151986.8711555, "finished": 1559151987.2974074}, null, "Traceback (most recent call last):\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/hpbandster/core/worker.py\", line 206, in start_computation\n    result = {'result': self.compute(*args, config_id=id, **kwargs),\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 475, in compute\n    _ = classifier.train(input_fn_train)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 363, in train\n    loss = self._train_model(input_fn, hooks, saving_listeners)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 843, in _train_model\n    return self._train_model_default(input_fn, hooks, saving_listeners)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 856, in _train_model_default\n    features, labels, model_fn_lib.ModeKeys.TRAIN, self.config)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 831, in _call_model_fn\n    model_fn_results = self._model_fn(features=features, **kwargs)\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 139, in __call__\n    train_op = self.create_train_op(model) if mode == tf.estimator.ModeKeys.TRAIN else None\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 190, in create_train_op\n    if model.config['lr_scheduler'] in [\"inv_exp_fast\", \"inv_exp_slow\"]:\nKeyError: 'lr_scheduler'\n"]
[[17, 0, 0], 5, {"started": 1559151987.301075, "submitted": 1559151987.3009808, "finished": 1559151988.1731708}, null, "Traceback (most recent call last):\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/hpbandster/core/worker.py\", line 206, in start_computation\n    result = {'result': self.compute(*args, config_id=id, **kwargs),\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 475, in compute\n    _ = classifier.train(input_fn_train)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 363, in train\n    loss = self._train_model(input_fn, hooks, saving_listeners)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 843, in _train_model\n    return self._train_model_default(input_fn, hooks, saving_listeners)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 856, in _train_model_default\n    features, labels, model_fn_lib.ModeKeys.TRAIN, self.config)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 831, in _call_model_fn\n    model_fn_results = self._model_fn(features=features, **kwargs)\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 139, in __call__\n    train_op = self.create_train_op(model) if mode == tf.estimator.ModeKeys.TRAIN else None\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 190, in create_train_op\n    if model.config['lr_scheduler'] in [\"inv_exp_fast\", \"inv_exp_slow\"]:\nKeyError: 'lr_scheduler'\n"]
[[18, 0, 0], 5, {"started": 1559151988.1818702, "submitted": 1559151988.1817522, "finished": 1559151988.7652287}, null, "Traceback (most recent call last):\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/hpbandster/core/worker.py\", line 206, in start_computation\n    result = {'result': self.compute(*args, config_id=id, **kwargs),\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 475, in compute\n    _ = classifier.train(input_fn_train)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 363, in train\n    loss = self._train_model(input_fn, hooks, saving_listeners)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 843, in _train_model\n    return self._train_model_default(input_fn, hooks, saving_listeners)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 856, in _train_model_default\n    features, labels, model_fn_lib.ModeKeys.TRAIN, self.config)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 831, in _call_model_fn\n    model_fn_results = self._model_fn(features=features, **kwargs)\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 139, in __call__\n    train_op = self.create_train_op(model) if mode == tf.estimator.ModeKeys.TRAIN else None\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 190, in create_train_op\n    if model.config['lr_scheduler'] in [\"inv_exp_fast\", \"inv_exp_slow\"]:\nKeyError: 'lr_scheduler'\n"]
[[19, 0, 0], 5, {"started": 1559151988.7686825, "submitted": 1559151988.7685785, "finished": 1559151989.2954712}, null, "Traceback (most recent call last):\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/hpbandster/core/worker.py\", line 206, in start_computation\n    result = {'result': self.compute(*args, config_id=id, **kwargs),\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 475, in compute\n    _ = classifier.train(input_fn_train)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 363, in train\n    loss = self._train_model(input_fn, hooks, saving_listeners)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 843, in _train_model\n    return self._train_model_default(input_fn, hooks, saving_listeners)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 856, in _train_model_default\n    features, labels, model_fn_lib.ModeKeys.TRAIN, self.config)\n  File \"/home/msaragoc/anaconda3/envs/py3.5/lib/python3.5/site-packages/tensorflow/python/estimator/estimator.py\", line 831, in _call_model_fn\n    model_fn_results = self._model_fn(features=features, **kwargs)\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 139, in __call__\n    train_op = self.create_train_op(model) if mode == tf.estimator.ModeKeys.TRAIN else None\n  File \"/home/msaragoc/Kepler_planet_finder/src_hpo/worker_tf_locglob.py\", line 190, in create_train_op\n    if model.config['lr_scheduler'] in [\"inv_exp_fast\", \"inv_exp_slow\"]:\nKeyError: 'lr_scheduler'\n"]
